docs/connecting-to-a-database.md

```diff
@@ -0,0 +1,97 @@
+# Connecting to a database
+
+> ⚠️ **Warning**: Particularly if you're connecting to a production database,
+> proceed with caution and look to pair with someone else.
+
+Using the AWS console, you can connect via a bastion host to the:
+
+- [PostgreSQL database](#connecting-to-the-postgresql-rds-postgresql-database)
+- [MongoDB database](#connecting-to-the-mongodb-documentdb-database)
+
+## Prerequisites
+
+- Access to the relevant AWS account for the environment
+
+## Environment information
+
+| Environment | AWS account       | Systems Manager key | EC2 jumpbox                      |
+|-------------|-------------------|---------------------|----------------------------------|
+| Staging     | StagingAPIs       | staging             | RDS jump box-Platform APIs (new) |
+| Production  | Mosaic-Production | mosaic-prod         | bastion-rds-jump-box-prod        |
+
+## Connecting to the PostgreSQL (RDS PostgreSQL) database
+
+1. Within relevant AWS account, go to **Systems Manager** → **Parameter Store**
+2. Search for **/social-care-case-viewer-api/\<environment-key\>/postgres**
```

Comment:
Is the Systems Manager key the name we expect to see when looking for the parameters?

Perhaps, this should be:
```suggestion
2. Search for **/social-care-case-viewer-api/\<systems-manager-key\>/postgres**
```

Reply:
Good spot, thanks!